Lombiq.NodeJs.Extensions/scripts/run-pnpm-script.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync, exec } = require('child_process');
+
+const { handleErrorObject } = require('./handle-error');
+
+function panic(message) {
+    handleErrorObject(message);
+    process.exit(1);
+}
+
+// Check if pnpm is installed.
+try { execSync('pnpm -v'); }
+catch (_) { panic('This project requires pnpm. You can install it with the "npm install --global pnpm" command.'); }
+
+// Load command line arguments.
+const args = process.argv.splice(2);
+if (args.length < 2) panic('USAGE: node scripts/run-pnpm-script project-path script');
```

Comment:
Let's mark parameters as such:
```suggestion
if (args.length < 2) panic('USAGE: node scripts/run-pnpm-script <project-path> <script-name>');
```